src/App.jsx

```diff
@@ -24,6 +25,9 @@ const translations = {
   ja: require("./translations/ja.json"),
 };
 
+var userLatitude = 0;
+var userLongitude = 0;
```

Comment:
Could you try and see if we can replace these with useState variables instead e.g.
```
  const [currentLatitude, setCurrentLatitude] = useState(null);
  ```